content-repo/extra-docs/articles/integrations-and-incident-health-check.md

```diff
@@ -0,0 +1,109 @@
+---
+id: integrations-and-incident-health-check
+title: Integrations and Incidents Health Check
+description: The Health Check for Integrations and Incidents content pack uses out-of-the-box playbooks, scheduled as a job, to check for, return, and display information about failed integrations and incidents with errors. As part of the playbook run, users will be sent an email notification when failed incidents and/or integrations are discovered.
+---
+ 
+
+## What's in this Content Pack?
+ 
+This pack includes the several content items that enable you to retrieve and view data about failed integrations and incidents. The out-of-the-box items are robust enough to get started with, but are easily customizable to fit your specific requirements.
+- [Playbooks](#Playbooks)
+- [Incident Layouts](#Incident-Layouts)
+- [Dashboards & Lists](#Dashboards-and-Lists)
+
+
+### Playbooks
+This pack contains a parent playbook that calls two sub-playbooks.
+ 
+| Playbook | Description | Notes |
+|---------------- | ------------- | ------------- |
+| JOB - Integrations and Playbooks Health Check | This is the pack's parent playbook and is the default playbook for the **Integrations and Incidents Health Check** incident type. You should run this playbook as a scheduled job. The playbook checks the health of all enabled integrations and open incidents | Under the playbook inputs, you can add recipients to send the health check report via email. See below for additional features of this playbook. |
+| Integrations and Playbooks Health Check - Running Scripts | This playbook is executed as part of the **JOB - Integrations and Playbooks Health** parent playbook and is responsible for running scripts that check the system for failed integrations and failed incidents.  | You can run the playbook on its own (not only as a sub-playbook of the parent playbook) to run health checks on enabled integrations and open incidents. |
+| JOB - Integrations and Playbooks Health Check - Lists handling | This playbook is executed as part of the **JOB - Integrations and Playbooks Health** parent playbook and is responsible for creating or updating related XSOAR lists. | - |
+ ---
+ 
+In addition to running the sub-playbooks, the parent playbook **JOB - Integrations and Playbooks Health Check** performs the following functions:
+- Send the health check report to the recipients from the playbook inputs.
+- If another health check investigation is open (incident, not a job), the playbook will:
+  - Check if analyst notes were added to the failed integrations and failed incidents grids and **copy** them to the new investigation.
+  - Link the old investigation to the new one and close the old investigation due to irrelevance.
+Re-run the health check tests. It is a manual action because you will want to re-run them only after fixing the issues, not before.
```

Comment:
i think we should add "-" before the "Re-run the health check tests. It is a manual.... "